docs/button.html

```diff
@@ -59,6 +59,21 @@ <h2>Components</h2>
                 </code>
             </pre>
 
+            <div>
+                <p>Link button</p>    
+                <div class="button">
```

Comment:
Giving a div a class name of button seems to be misleading.  If possible provide a different name for this class